samples/SdkTypeExtensions/Blob/BlobInputBindingSamples.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
```

Comment:
need samples binding trigger and input to POCO. I know you have the Book sample that does trigger. Maybe your trigger/input samples should also show pocos.

Are there any regression tests for this stuff, or is it only in samples that have to be manually run?

Reply:
I'm not sure, @kshyju @fabiocav do you know if we have regression tests in place for the dotnet worker and extensions?